fmriprep/run_workflow.py

```diff
@@ -121,8 +124,7 @@ def create_workflow(opts):
     # Set nipype config
     ncfg.update_config({
         'logging': {'log_directory': log_dir, 'log_to_file': True},
-        'execution': {'crashdump_dir': log_dir, 
-                      'remove_unnecessary_outputs': False}
+        'execution': {'remove_unnecessary_outputs': False}
```

Comment:
Had removed it since crashdumpdir is being set in base.py for every node. I'll re-add it as a fallback in case the new way of setting the crash directory fails for any reason we can consistently find crash files and know that the directory wasn't set to the subject directory.